### PR TITLE
Add registration link for Ghent local site

### PR DESCRIPTION
**Pull request by nvnieuwk**

### CODE REVIEW

File: src/content/events/2024/hackathon-march-2024/belgium-ghent-university.md [hunk 1]
```diff
@@ -7,22 +7,24 @@ start_time: '10:00+02:00'
...
+address: Hermann von Helmholtz, campus Ardoyen, iGent, 9052 Zwijnaarde, Belgium
 location_url: https://www.ugent.be/en
-# location_latlng: [] // TODO Fill this in once known
+location_latlng: [51.01287821684896, 3.708004390525348]
 ---
 
 Local event to be held at Ghent University, Belgium
 
+Registrations are open! Register [here](https://event.ugent.be/registration/event/138e28ec-3e42-4370-833d-628d578e047c) to secure your spot.
+
```
Comment (mashehu) [inline src/content/events/2024/hackathon-march-2024/belgium-ghent-university.md:18]: We will have a centralized registration, where we will forward you the signups to Ghent. so no need for you to handle it
```suggestion
```

**Review by mashehu — CHANGES_REQUESTED** [review]

File: src/content/events/2024/hackathon-march-2024/belgium-ghent-university.md (hunk 1 above)
Comment (nvnieuwk) [reply src/content/events/2024/hackathon-march-2024/belgium-ghent-university.md:18]: Oh okay good to know, thanks :) when will this be available?

Comment (mashehu) [reply src/content/events/2024/hackathon-march-2024/belgium-ghent-university.md:18]: we open the registration on february 2nd. you can ping @christopher-hakkaart if you need info at a certain date.

Comment (nvnieuwk) [reply src/content/events/2024/hackathon-march-2024/belgium-ghent-university.md:18]: Thank you! February 2nd is perfect :)

**Review by mashehu — APPROVED** [review]

**Pull request merged**